homeassistant/components/vacuum/services.yaml

```diff
@@ -131,3 +131,19 @@ xiaomi_remote_control_move_step:
     duration:
       description: Duration of the movement.
       example: '1500'
+
+xiaomi_clean_zone:
+  description: Start the cleaning operation in the selected areas for the number of repeats indicated.
+  fields:
+    entity_id:
+      description: Name of the vacuum entity.
+      example: 'vacuum.xiaomi_vacuum_cleaner'
+    zone:
+      description: Array of zones. Each zone is an array of 4 integer values.
+      example: '[[23510,25311,25110,26361]]'
```

Comment:
It would be nice to have an explanation on how to get the zones.

Reply:
You can use the Flolevac app for Android (https://play.google.com/store/apps/details?id=de.flole.xiaomi).
After installing the app, clean the whole house (to create the complete map).
Press clean spot, select area, long press clean button and zone coordinates are pasted to clipboard.